packages/wrapped-keys/src/lib/constants.ts

```diff
@@ -0,0 +1,13 @@
+const CHAIN_ETHEREUM = 'ethereum';
+const LIT_PREFIX = 'lit_';
+// Update the endpoint to Wrapped Key project endpoint
+const ENCRYPTED_PRIVATE_KEY_ENDPOINT =
+  'https://8wugrwstu1.execute-api.us-east-2.amazonaws.com/encrypted';
```

Comment:
Should we make this configurable? We should also add some docs on what the different aws envs are.

Reply:
@MaximusHaximus

Reply:
He's working on a PR to do this

Reply:
https://github.com/LIT-Protocol/js-sdk/pull/513